packages/components/src/styles/interactive/interactive.stories.ts

```diff
@@ -64,88 +36,41 @@ export const Default = {
 };
 
 /**
- * Use the `disabled` class to make an interactive element look disabled. This works as well when setting an `disabled` attribute on the element.
+ * Use the `sd-interactive--reset` class to reset the default browser styles of e. g. a button.
  */
 
-export const Disabled = {
-  parameters: { controls: { exclude: ['sd-interactive--disabled', 'sd-interactive--reset'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: [{ type: 'attribute', name: 'sd-interactive--disabled', values: [false, true] }]
-      },
-      options: { templateContent: '<button class="%CLASSES%">%SLOT%</button>' },
-      constants: { type: 'attribute', name: 'sd-interactive--disabled', value: true },
-      args
-    });
-  }
+export const Reset = {
+  render: () => html`
+    <div class="flex flex-row gap-12">
```

Comment:
Since we only have a button now I think you can remove the wrapper div

Reply:
Updated